### PR TITLE
[EI-51] Add docs for interest functions, and new interest filters

### PR DESCRIPTION
**Pull request by jspellman814**

### CODE REVIEW

File: docs/interest.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+$interest = Interest\get_interest( $data );
+```
+
+### `set_interest`
+
+Set the interest header key and data.
+
+#### Parameters
+
+$key __(array)__ Key for the header, or array of keys.
```
Comment (jazzsequence) [inline docs/interest.md]: I would wrap variable names in backticks, e.g.:
```suggestion
`$key` __(array)__ Key for the header, or array of keys.
```
🔢

File: docs/interest.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+}
+```
+
+### `pantheon.ei.interest_threshold`
+
+Modify the interest threshold. Default `3`.
+
+#### Parameters
+
+__(int)___ The interest threshold.
```
Comment (jazzsequence) [inline docs/interest.md]: One too many underscores

```suggestion
__(int)__ The interest threshold.
```

**Review by jazzsequence — APPROVED** [review]
Couple minor tweaks. Preemptively approving.

File: docs/interest.md (hunk 2 above)
Comment (jspellman814) [reply docs/interest.md]: Addressed [93e7991](https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/pull/5/commits/93e79914fef2f9bfa99c1eacc9cba00222cfc0c4)

File: docs/interest.md (hunk 1 above)
Comment (jspellman814) [reply docs/interest.md]: Addressed [93e7991](https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/pull/5/commits/93e79914fef2f9bfa99c1eacc9cba00222cfc0c4)

**Pull request merged**